style.css

```diff
@@ -1,825 +1,810 @@
   }
 
-  .first__sidebar-social {
-    width: 92px;
-    padding: 490px 42px 0;
-  }
-}
-
-@media (max-width: 750px) {
-  .header__language,
-  .header__vector,
-  .first__sidebar,
-  .first__collection,
-  .collection__scroll,
-  .collection__scroll-elem,
-  .img__img,
-  .text__subtitle,
-  .collection__next-text,
-  .collection__other-text,
-  .collection__vector {
-    display: none;
+  .second__bag-dsk {
+    display: block;
+    margin-top: 56px;
+    width: 500px;
   }
 
-  .first-container {
-    padding: 339px 0;
-    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
-    background-size: 750px;
-    background-position: center;
+  .second__box {
+    width: 393px;
+    height: 446px;
   }
 
-  .header__inner {
-    width: 100%;
-    height: 140px;
-    background: #fff;
-    z-index: 2;
+  .second__title {
+  	font-size: 32px;
+  	line-height: 1;
+  	padding-top: 10px;
+  	margin-bottom: 13px;
   }
 
-  .header__menu {
-    background: #ededed;
-    z-index: 3;
-    height: 140px;
-    width: 140px;
+  .second__container {
+  	width: 306px;
+  	height: 348px;
   }
 
-  .header__menu-item {
-    height: 4px;
-    width: 48.7px;
-    margin-bottom: 10px;
+  .second__subtitle {
+    font-size: 13px;
+    margin-bottom: 46px;
   }
 
-  .header__logo p {
-    margin: 25px 21px;
-    color: black;
-    font-size: 60px;
-    letter-spacing: 4px;
+  .second__text {
+  	font-size: 12px;
+  	margin-bottom: 32px;
   }
 
-  .text__title {
-    position: inherit;
-    align-items: flex-end;
-    font-size: 98px;
-    line-height: 103px;
-    padding-left: 52px;
-    letter-spacing: 5px;
+  .second__block-color-gray,
+  .second__block-color-pink,
+  .second__block-color-blue,
+  .second__block-color-green {
+  	width: 24px;
+    height: 24px;
   }
 
-  .text__container {
-    display: flex;
-    align-items: flex-end;
-    width: 100%;
-    max-width: 750px;
-    height: 945px;
-    letter-spacing: 5px;
+  .second__block-color-pink {
+	  margin-left: 11.82px;
   }
 
-  .step {
-    display: flex;
+  .second__block-color-blue {
+    margin-left: 11.82px;
   }
 
-  .text {
-    width: 100%;
-    max-width: 750px;
-    height: 600px;
+  .second__block-color-green {
+    margin-left: 11.82px;
   }
 
-  .vectormob {
-    position: absolute;
-    width: 67px;
-    height: 39px;
-    left: 345px;
-    top: 1016px;
-    display: block;
+  .second__block-coust {
+  	font-size: 34px;
+  	margin-right: 20px;
   }
-}
 
-@media (max-width: 660px) {
-  .first-container {
-    padding: 267px 0;
-    background: url(img/pete-bellis-432867-unsplashmob.png) no-repeat;
-    background-size: 660px;
-    background-position: center;
+  .second__quantity {
+  	text-decoration: none;
+    color: #000;
+    display: block;
+    font-size: 15px;
+    border: 1.5px solid #000;
+    padding: 7px 25px;
+    margin-right: 9px;
+    margin-left: 44px;
   }
-
-  .header__inner {
-    width: 100%;
-    height: 123px;
-    background: #fff;
-    z-index: 2;
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/step3/pullRequest?prid=5898150)